Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSLoginWindowController.mm

```diff
@@ -116,7 +116,7 @@ - (void)save
 {
     // windows segments by product name here. in since user defaults belong to this product, we
     // don't need to do that.
-    NSString* serverName = [NSString stringWithSTString:GetServerDisplayName()];
+    NSString* serverName = ST2NSString(GetServerDisplayName());
```

Comment:
I think this is a memory leak because ST2NSString is no longer ARC managed and returns a +1 CFString. Would prefer if the Obj-C interface was left operational to solve this issue.